### ISSUE TITLE
Update check prompt only appears on second run

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/dangerzone#512]
# Steps to reproduce

Install Dangerzone 0.4.2 on Mac

# Expected Behavior

Dangerzone prompts me if I want to enable automatic update checks

# Actual behavior

Prompt only appears the second time I run Dangerzone

**Comment by deeplow** [comment]
Yes, this is expected behavior. On the first run we assume that Dangerzone is already up to date and in case there's ever an onboarding flow, this way it doesn't conflict with that.

Our assumption is that this won't really confuse users. Do you think it may?

**Comment by eloquence** [comment]
Thanks, since it's expected behavior I've removed the `bug` label and added `ux` to discuss a bit further as we dig into overall UX.

I think it is potentially a bit confusing if the user discovers the "Check for updates" switch in the menu before the prompt runs. It'd be a bit odd to change that setting through the menu, and to then be prompted on the second run whether you want to check for updates.

**Comment by deeplow** [comment]
Good point!

**Comment by apyrgio** [comment]
Regarding this last part:

> It'd be a bit odd to change that setting through the menu, and to then be prompted on the second run whether you want to check for updates.

Note that the user will not be prompted unconditionally on the second run. They will only be prompted if they haven't expressed their preference before (including toggling the "Check for updates" button). So we should be ok in this scenario.

+1 for the UX consideration. I basically copied what other update frameworks do, that's why I chose to prompt users on the second run. We can discuss if this approach should apply to Dangerzone as well.
